commands/lock.js

```diff
@@ -0,0 +1,105 @@
+const util = require('../lib/util.js');
+const Discord = require('discord.js');
+
+const command = {};
+
+command.description = 'Dissallow users to send messages to a channel';
+
+command.usage = '<global|#channel|id…> message';
+
+command.names = ['lock'];
+
+command.execute = async (message, args, database, bot) => {
+  //Permission check
+  if (!util.isMod(message.member) && !message.member.hasPermission('MANAGE_CHANNELS')) {
+    await message.react(util.icons.error);
+    return;
+  }
+
+  let channels = await util.channelMentions(message.guild,args);
+  let embed = new Discord.MessageEmbed().setTitle('This channel has been locked!').setDescription(args.join(' ')).setColor(util.color.red);
```

Comment:
```suggestion
  let embed = new Discord.MessageEmbed().setTitle('This channel is locked.').setDescription(args.join(' ')).setColor(util.color.red).setFooter('You are not muted, this channel is locked for everyone. Don't send direct messages to team members or moderators .');
```
(Or a similar footer text, not sure about the exact wording yet.)